src/fn/upload-custom-translations.js

```diff
@@ -23,6 +23,11 @@ module.exports = () => {
         .map(fileName => {
           const id = idFor(fileName);
           const languageCode = id.substring('messages-'.length);
+          const languageCodeIsValid = isLanguageCodeValid(languageCode);
+          if (!languageCodeIsValid) {
```

Comment:
nit: you could eliminate this variable since the name of the function already describes the code's intent